packages/react-native-reanimated/src/logger/logger.ts

```diff
@@ -2,6 +2,10 @@
 import { addLogBoxLog } from './LogBox';
 import type { LogData, LogBoxLogLevel } from './LogBox';
 
+const DOCS_URL =
+  'https://docs.swmansion.com/react-native-reanimated/docs/debugging/logger-configuration';
+const DOCS_REFERENCE = `If you don't want to see this message, you can disable the \`strict\` mode. Refer to:\n${DOCS_URL} for more details.`;
+
```

Comment:
This is related to docs and includes a link to the documentation page so I would prefer to merge this change together with docs.